src/components/Navbar.tsx

```diff
@@ -22,59 +22,74 @@ function Navbar({ scrollToFooter }: { scrollToFooter: any }) {
   const location = useLocation();
 
   const props = useSpring({
-    loop: true,
+    to: async (next) => {
+      while (true) {
+        await next({
+          opacity: 1,
+          boxShadow: "0px 0px 5px rgba(255, 255, 255, 1)",
+        });
+        await new Promise((resolve) => setTimeout(resolve, 1000));
+        await next({
+          opacity: 0.9,
+          boxShadow: "0px 0px 0px rgba(255, 255, 255, 0)",
+        });
+        await new Promise((resolve) => setTimeout(resolve, 4000));
+      }
+    },
```

Comment:
Can you make sure you are working off of the latest main? I don't think this change should be here.